src/plugins/condition/components/Condition.vue

```diff
@@ -42,28 +42,15 @@
 
         <span class="c-condition__name">{{ condition.configuration.name }}</span>
         <!-- TODO: description should be derived from criteria -->
-        <span v-if="condition.isDefault"
-              class="c-condition__summary"
-        >
-            When all else fails
-        </span>
-        <span v-else
-              class="c-condition__summary"
-        >
+        <span class="c-condition__summary">
             <template v-if="!canEvaluateCriteria">
                 Define criteria
             </template>
-            <template v-else>
-                When
-                <span v-for="(criterion, index) in condition.configuration.criteria"
-                      :key="index"
-                >
-                    {{ getRule(criterion, index) }}
-                    <template v-if="!isLastCriterion">
-                        {{ getConjunction }}
-                    </template>
-                </span>
-            </template>
+            <span v-else>
+                <condition-description :show-label="false"
```

Comment:
good choice to make as its own component. we just need to be sure the styling remains the same. the description component is inline in read-only mode and as a block element in edit mode.

Reply:
@charlesh88 Can you take a look at the CSS here?

Reply:
And I'm not trying to say its not working. Just making sure this was verified. The markup was slightly different. One block used divs and one used spans. Maybe the container elements take care of the proper displays.